scripts/testnet/include/config.sh

```diff
@@ -106,12 +107,30 @@ updateElasticsearchIndices() {
   popd
 }
 
+updateConfigFile() {
+  if [ "$ADDRESS_HRP" == "erd" ]; then
+    return 1
+  fi
+
+  file="$1"
+
+  grep -o '"erd1[^"]*"' "$file" | tr -d '"' | sort -u | while read -r addr; do
```

Comment:
We should make sure somehow that every string that starts with erd1 is actually an address.
What if it's something else and the argument you send to the script is invalid?

Reply:
I changed to `"erd1` and `len=62`